test/test_mqtt5.py

```diff
@@ -1549,6 +1549,132 @@ def test_operation_statistics_uc1(self):
         client.stop()
         callbacks.future_stopped.result(TIMEOUT)
 
+    # ==============================================================
+    #             5to3 ADAPTER TEST CASES
```

Comment:
It would be nice if we could find a way to reuses the tests, while it is a little tricky here as the client is actually setup as mqtt5 client. The tests should need to combine the mqtt5 setup and the operations through mqtt3 interface.